### PR TITLE
[FLINKCC-364] Add a time buffer to the token validator

### PR DESCRIPTION
**Pull request by pgaref**
Release Notes
-------------

Bug Fixes
- https://confluentinc.atlassian.net/browse/FLINKCC-364

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * **yes**: ok

What
----

https://confluentinc.atlassian.net/browse/FLINKCC-364

References
----------

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

**Review by YannickPferr — APPROVED** [review]

**Comment by pgaref** [comment]
@sgagniere can you please take a look?

File: pkg/cmd/token_validator.go [hunk 1]
```diff
@@ -48,7 +48,8 @@ func (v *JWTValidatorImpl) Validate(context *config.Context) error {
 		return errors.New("malformed token: no expiration")
 	}
 
-	if float64(v.Clock.Now().Unix()) > exp {
+	// Add a time buffer of 1 minute to the token validator
+	if float64(v.Clock.Now().Unix()+60) > exp {
```
Comment (brianstrauch) [inline pkg/cmd/token_validator.go:52]: Add `time.Minute` to the date instead of 60? (and then call `.Unix()`)

Comment (sgagniere) [reply pkg/cmd/token_validator.go:52]:
```suggestion
	if float64(v.Clock.Now().Add(time.Minute).Unix()+60) > exp {
```
Based on Brian's suggestion. You'll also need to import `"time"` at the top.

**Review by sgagniere — APPROVED** [review]

**Comment by YannickPferr** [comment]
@brianstrauch @sgagniere could we please do one more release with this change?

**Comment by brianstrauch** [comment]
How did we decide on one minute here @pgaref?

**Comment by pgaref** [comment]
> How did we decide on one minute here @pgaref?

@brianstrauch  more context under https://confluent.slack.com/archives/C059W6DU2U9/p1695506661440219?thread_ts=1695501483.707299&cid=C059W6DU2U9

**Pull request merged**